test/unit/exception_handling_test.rb

```diff
@@ -659,10 +659,23 @@ def to_s
                    "/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/test/unit/autorunner.rb:12:in `run'",
                    "/System/Library/Frameworks/Ruby.framework/Versions/1.8/usr/lib/ruby/1.8/test/unit.rb:279",
                    "-e:1"]
+
+      module ::Rails
+        class BacktraceCleaner
+          def clean(_backtrace)
+            []
+          end
+        end
+      end
+
+      mock(Rails).backtrace_cleaner { Rails::BacktraceCleaner.new }
+
       ex = Exception.new
       ex.set_backtrace(backtrace)
       result = ExceptionHandling.send(:clean_backtrace, ex)
       assert_equal backtrace, result
+
+      Object.send(:remove_const, :Rails)
     end
```

Comment:
I don't think this is going to be a problem here, but it is a best practice.  Can you undefine this module after your test?

```
ensure
  Object.send(:remove_const, :Rails)
```

That will prevent phantoms